requirements/pip-tools.txt

```diff
@@ -12,7 +12,7 @@ packaging==21.3
     # via build
 pep517==0.13.0
     # via build
-pip-tools==6.9.0
+pip-tools==6.10.0
```

Comment:
No breaking change mentioned in the new version changelog: https://github.com/jazzband/pip-tools/blob/main/CHANGELOG.md#6100-2022-11-13